<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpcionActualizar</class>
 <widget class="QWidget" name="OpcionActualizar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelTitulo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>OPCIONES ACTULIZAR ADMIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelActU">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Actulizar Usuario</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAcN">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Actulizar Negocio</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAcP">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Actulizar Producto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonAcU">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonAcN">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonAcP">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonVolver">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
